cmd/tsbs_generate_queries/databases/mongo/finance.go

```diff
@@ -0,0 +1,437 @@
+package mongo
+
+import (
+	"encoding/gob"
+	"fmt"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/finance"
+	"github.com/timescale/tsbs/pkg/query"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func init() {
+	gob.Register(bson.A{})
+}
+
+type Finance struct {
+	*BaseGenerator
+	*finance.Core
+}
+
+func (f *Finance) LastPrice(q query.Query) {
+	query := q.(*query.Mongo)
+	query.Pipeline = mongo.Pipeline{
+		{
+			{"$sort", bson.D{
+				{"time", 1},
+			}},
+		},
+		{
+			{"$group", bson.D{
+				{"_id", "$tags.symbol"},
+				{"lastPrice", bson.D{
+					{"$last", "$price"},
+				}},
+			}},
+		},
+	}
+	query.CollectionName = []byte("point_data")
+	query.HumanLabel = []byte("MongoDB last price per symbol")
+	query.HumanDescription = query.HumanLabel
+}
+
+func (f *Finance) OpenHighLowClose(q query.Query, interval time.Duration) {
```

Comment:
Given the query is hard coded with unit "minutes", I think it can just take a number here.

Reply:
That also makes sense